tests/widget/operatorWithValidator/dashboard/bondRewards.spec.ts

```diff
@@ -0,0 +1,289 @@
+/* eslint-disable no-irregular-whitespace */
+import { COMMON_ACTION_TIMEOUT } from 'tests/consts/timeouts';
+import { expect } from '@playwright/test';
+import { test } from '../../test.fixture';
+import { qase } from 'playwright-qase-reporter/playwright';
+import { CSAccountingContract } from 'tests/services/csmContracts.service';
+import { formatEther } from '@ethersproject/units';
+import { USD_AMOUNT_REGEX } from 'tests/consts/regexp.const';
+
+test.describe('Dashboard. Bond & Rewards.', async () => {
+  test.beforeEach(async ({ widgetService }) => {
+    await widgetService.connectWallet();
+    await widgetService.dashboardPage.open();
+  });
+
+  test(
+    qase(134, 'Available to Claim Section Verification'),
+    async ({ widgetService }) => {
+      const availableToClaim =
+        widgetService.dashboardPage.bondRewards.availableToClaim;
+
+      await test.step('Wait for balance loaded', async () => {
```

Comment:
I suggest to create functions `getAvailableToClaimBalance` with `waitForTextContent` to reduce flaky balance getter. It would help to not depend on custom `Wait for balance loaded` since:
- default balance refetch will flaky the once once with value = ''( during fetch).